src/scripts/install.sh

```diff
@@ -11,11 +15,20 @@ Install() {
 
     BINPATH="${HOME}/.dctlenv/bin"
     if [ ! -d "${HOME}/.dctlenv" ]; then
-      git clone --depth 1 --branch v0.1.8 https://github.com/wbeuil/dctlenv ~/.dctlenv
+      git clone --depth 1 --branch v0.1.9 https://github.com/wbeuil/dctlenv ~/.dctlenv
     fi
 
     VERSION=$(GetVersion)
-    gpg --import driftctl_pubkey.pem
+
+    # Since CircleCI breach on 05/01/23, driftctl changed its signing key
+    # which means the PGP signature will be different for versions below 0.38.1
+    driftctl_key="65DDA08AA1605FC8211FC928FFB5FCAFD223D274"
+    if VersionLowerOrEqual "${VERSION/v/}" "0.38.1"; then
+      driftctl_key="277666005A7F01D484F6376DACC776A79C824EBD"
+    fi
```

Comment:
This check logic is already implemented within dctlenv, we could simplify this script by just providing the two keys to the gpg agent and then let dctlenv use the good one.